binlog_streamer.go

```diff
@@ -261,7 +268,8 @@ func (s *BinlogStreamer) handleRowsEvent(ev *replication.BinlogEvent) error {
 		return nil
 	}
 
-	dmlEvs, err := NewBinlogDMLEvents(table, ev, pos)
+	s.lastResumableBinlogPosition.Name = s.lastStreamedBinlogPosition.Name
```

Comment:
Wait, what? I don't think this should be legal 😄 jokes aside, mutating any binlog position sounds like a recipe for bugs down the line.. it's a bit like randomizing the coordinates you put in a GPS.. you might end up in middle of nowhere!

 Would a test fail if I removed this line?

Reply:
I wouldn't recommend removing it 😄. 

Instead of just duplicating all the places where `lastStreamedBinlogPosition` is set to also set this value, we just reuse the value like is already being done for the `pos` value in line 262. 

Do you think it makes more sense to put it closer to the `pos` assignment? I don't think it would make much different

Reply:
Oh I see what's going on. This could use some cleanup, I'll PR something based off your branch.

Reply:
Ended up moving this assignment to the `RotateEvent` and in the initial connect further up. That allows us to remove it from this line here.